components/registration/vaccination-modal.html

```diff
@@ -0,0 +1,20 @@
+<div class="modal-header page">
+    <h3>Vaksineringer</h3>
+</div>
+<div class="modal-body page">
+    <table class="dhis2-list-table-striped default-form-table">
+        <tr ng-repeat="vaccine in gridData">
+            <td>{{dateFromItem(vaccine)}}<br>
+                {{vaccine.recorder.name}}<div ng-if="!vaccine.performedByRecorder"><br>(Utført av en annen helseenhet)</div></td>
+            <td><div style="padding-bottom:5px">{{vaccine.atcCode}}({{vaccine.lotNumber}})
+        </tr>
+    </table>
+    <div ng-if="!gridData || gridData.length == 0">
```

Comment:
FHI returns kanLevereUtData=false to mark that the organisation making the request is not authorised to view data on the provided pid. Id est; you can not make the assumption that no gridData = no vaccination without also checking on that boolean field.